### PR TITLE
feat(tests): Add unit-tests to the components

### PR DESCRIPTION
**Pull request by bioN1ck**

### CODE REVIEW

File: src/components/Counter/Counter.test.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import Counter from './Counter';
+
+describe('Counter', () => {
+  it('should render default value', async () => {
```
Comment (melk0sha) [inline src/components/Counter/Counter.test.tsx:6]: It's actually not required to use `async` and `await` for such cases as it should work without them

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
     "classnames": "^2.3.2",
+    "jest": "^27.5.1",
```
Comment (melk0sha) [inline package.json:15]: It would be nice to have `cypress` to fully complete the task 🙂
https://www.cypress.io/

Comment (bioN1ck) [reply package.json:15]: Added

File: src/components/Counter/Counter.test.tsx (hunk 1 above)
Comment (bioN1ck) [reply src/components/Counter/Counter.test.tsx:6]: It's strange, but specifically for this component 'screen' function returns a Promise.

**Review by melk0sha — APPROVED** [review]

**Pull request merged**